Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.py

```diff
@@ -1536,6 +1676,14 @@ def test_module():
         demisto.results(get_table_name_command())
     if demisto.command() == 'servicenow-get-ticket-notes':
         demisto.results(get_ticket_notes_command())
+    if demisto.command() == 'servicenow-items-query':
+        demisto.results(query_items())
+    if demisto.command() == 'servicenow-item-details-get':
```

Comment:
Why not `servicenow-get-item-details`?

Reply:
this is the design i've got:
https://github.com/demisto/etc/issues/22557#issuecomment-603766282